src/elements/ProgressBar/ProgressBar.js

```diff
@@ -110,6 +117,31 @@ export default class ProgressBar extends React.Component {
 		this.interval = setInterval(this.timer, (segment.duration / segment.percentage));
 	}
 
+	handleKeyDown(event) {
+		if(event.key === 'Control') {
+			this.setState({
+				pressedControlKey: true
+			}, this.skipProgressBar);
+		}
+
+		if(event.key === 'k') {
+			this.setState({
+				pressedKKey: true
+			}, this.skipProgressBar);
+		}
+	}
+
+	skipProgressBar() {
+		if(this.state.pressedKKey && this.state.pressedKKey) {
```

Comment:
pressKKey is here twice. did you mean pressedControlKey?

Reply:
ooops! good catch